backend/public-graph/graph/resolver.go

```diff
@@ -2963,9 +2964,9 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 	updateSpan, _ := tracer.StartSpanFromContext(ctx, "public-graph.pushPayload", tracer.ResourceName("doSessionFieldsUpdate"))
 	defer updateSpan.Finish()
 
-	excluded := r.isSessionUserExcluded(ctx, sessionObj)
+	excluded, reason := r.isSessionExcluded(ctx, sessionObj, sessionHasErrors)
 	if excluded {
-		log.WithContext(ctx).WithFields(log.Fields{"session_id": sessionObj.ID, "project_id": sessionObj.ProjectID, "identifier": sessionObj.Identifier}).Infof("excluding session due to excluded identifier")
+		log.WithContext(ctx).WithFields(log.Fields{"session_id": sessionObj.ID, "project_id": sessionObj.ProjectID, "reason": *reason}).Infof("excluding session")
```

Comment:
should be fine to log this. if the spam becomes a problem, we can clean this up.
guessing you removed the the `sessionObj.Identifier` logging because we also log it in `isSessionExcluded`?